src/systems/tracked-controls.js

```diff
@@ -27,9 +29,7 @@ module.exports.System = registerSystem('tracked-controls', {
     }
   },
 
-  tick: function (time) {
-    if (time < this.lastControllersUpdate + 10) { return; }
-    this.lastControllersUpdate = time;
+  throttledTickHandler: function (time, delta) {
```

Comment:
This doesn't actually need to be renamed right? In `init`: `this.tick =utils.throttle(utils.bind(this.tick, this))`

Might also be useful to allow the throttleTick do binding for you since we're not using closures: ```utils.throttleTick(this.tick, 10, this);```